etc/rh-license-names.json

```diff
@@ -203,6 +203,20 @@
       "http://www.eclipse.org/org/documents/epl-v10.php"
     ]
   },
+  {
+    "name": "Eclipse Public License - v 2.0",
+    "url": "http://www.eclipse.org/legal/epl-v20.html",
+    "aliases": [
+      "Eclipse Public License, Version 2.0",
+      "Eclipse Public License 2.0",
+      "Eclipse Public License v2.0",
+      "Eclipse Public License, Version 2.0",
+      "Eclipse Public License (EPL), Version 2.0"
+    ],
+    "urlAliases": [
+      "http://www.eclipse.org/legal/epl-v20.html"
```

Comment:
Just to be on a safe side in the future could also add http://www.eclipse.org/legal/epl-2.0 to urlAliases. It's the one that was used in the dependency that caused a QE failure.

Reply:
Sorry just noticed it now

Reply:
Coolio, fixed :)